custom-sign-in/OktaNativeLogin/UserProfile/UserProfileViewController.swift

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2019 Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import OktaAuthSdk
+import OktaOidc
+import SVProgressHUD
+
+class UserProfileViewController: AuthBaseViewController {
+    
+    var successStatus: OktaAuthStatusSuccess?
+    var oidcStateManager: OktaOidcStateManager?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        successStatus = status as? OktaAuthStatusSuccess
+        titleLabel.text = "Welcome, \(successStatus?.model.embedded?.user?.profile?.firstName ?? "-")"
+        subtitleLabel.text = successStatus?.model.embedded?.user?.profile?.login
+        timezoneLabel.text = successStatus?.model.embedded?.user?.profile?.timeZone
+        localeLabel.text = successStatus?.model.embedded?.user?.profile?.locale
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
```

Comment:
Why is this needed? Maybe move this part to viewDidAppear?

Reply:
`viewDidAppear` can be called by iOS multiple times. I need to run this code exactly once
One second delay is intentional, I want to show basic information on the screen and then request access token